### DISCUSSION TITLE
[list view]sort icons inconsistency

### DISCUSSION THREAD

**Discussion by junlincc** [apache/superset#22478] [Ideas]
Sort icons should be displayed on Columns - Charts, Visualization Type, Dataset, Modified By, Last Modified, Created By
Im not sure missing on `Dataset` and `Created` By is by design 

<img width="892" alt="Screen Shot 2020-12-04 at 9 49 20 AM" src="https://user-images.githubusercontent.com/67837651/101196773-0706d200-3616-11eb-88cd-8b6171025ed9.png">

related issue: 
https://github.com/apache/incubator-superset/issues/11689#issue-742170841

**Comment by nytai** [comment]
We can probably add sorting on "Created by" but "Dataset" is by design. There some technical reasons related to the weird dataset relationship in charts that doesn't allow for us to sort on it.

**Comment by rusackas** [comment]
While this is a bit old, it still seems things are in this state, so the question remains relevant. That said, it's not a bug, so I'll move it to an Ideas discussion. CC @jess-dillard @kasiazjc in case you want to advocate for this one.
